css/styles.css

```diff
@@ -79,8 +83,90 @@ form {
   padding-bottom: 1rem;
 }
 
+ul {
+  list-style: none;
+  width: 100%;
+  padding: 0;
+}
+
+li {
+  width: 100%;
+  padding: 1rem 0;
+}
+
+.entries-nav {
+  color: white;
+  text-decoration: none;
+}
+
+.entries-header {
+  justify-content: space-between;
+  align-items: center;
+  font-family: 'Raleway', sans-serif;
+  font-size: 24px;
+  padding: 1rem 0;
+}
+
+.new-entry-button {
+  font-size: 16px;
+  width: 70px;
+  height: 30px;
+  border-radius: 50px;
+  border: none;
+  outline: none;
+  background-color: #edf1f7;
+  box-shadow: 1px 2px 3px rgba(99, 99, 99, 0.2);
+  cursor: pointer;
+  transition: background-color 0.2s ease-in-out;
+}
+
+.new-entry-button:hover {
+  background-color: #590202;
+  color: white;
+}
+
+.close-button {
+  width: 40px;
+  height: 40px;
+  border-radius: 20px;
+  border: none;
+  outline: none;
+  background-color: #edf1f7;
+  box-shadow: 1px 2px 3px rgba(99, 99, 99, 0.2);
+  cursor: pointer;
+  transition: background-color 0.2s ease-in-out;
```

Comment:
See if you can consolidate a bit of your button styles into a `button {}` CSS rule to remove some duplication around `cursor` and `transition` etc. D.R.Y. CSS is a Good Thing &trade; dattebayo 😄 

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Consolidated button styling. See commit changes